using-funds/cash-advances.md

```diff
@@ -0,0 +1,9 @@
+# Cash Advances
+
+After being approved for a [grant](https://github.com/bridgefoundry/operations/tree/master/grants), Independent Communities that can't afford to get reimbursed after the fact can receive funds via a cash advance.
+
+The big downside of a cash advance is that you will need to send back any extra money that you did not spend. If you can afford to do so, it is best to low ball your estimate and then get reimbursed for a the small amount that you go over.
+
+If you opt to receive a cash advance, you must **submit all receipts from the workshop within 7 days of the workshop and must follow all of our receipt policies exactly**, especially including itemization and proof of payment for each expense. Failure to do so or to return unused funds will prevent you from being able to receive another cash advance for future workshops.
```

Comment:
Then we would need to be extremely clear on what exactly the receipt policy is.

Reply:
Good point. I meant to link to our policies (probably this page: https://github.com/bridgefoundry/operations/blob/master/using-funds/README.md) - not sure if I can change that in the submitted code now or if I need to wait till the pull request is approved and then change it?

Reply:
You can make a change to this branch and push it.  It'll show up and the review continues.  That's often how people handle feedback from the reviews.